scripts/rollup/packaging.js

```diff
@@ -88,6 +94,9 @@ function getBundleOutputPath(bundleType, filename, packageName) {
         default:
           throw new Error('Unknown RN package.');
       }
+    case IIFE_DEV:
+    case IIFE_PROD:
+      return `build/node_modules/${packageName}/iife/${filename}`;
```

Comment:
Similar here: let's remove the `iife` subdirectory and place it directly at the top level.